src/script/ismine.h

```diff
@@ -9,22 +9,48 @@
 #include <script/standard.h>
 
 #include <stdint.h>
+#include <bitset>
 
 class CKeyStore;
 class CScript;
 
 /** IsMine() return codes */
 enum isminetype
 {
-    ISMINE_NO = 0,
-    ISMINE_WATCH_ONLY = 1,
-    ISMINE_SPENDABLE = 2,
-    ISMINE_ALL = ISMINE_WATCH_ONLY | ISMINE_SPENDABLE
+    ISMINE_NO         = 0,
+    ISMINE_WATCH_ONLY = 1 << 0,
+    ISMINE_SPENDABLE  = 1 << 1,
+    ISMINE_ALL        = ISMINE_WATCH_ONLY | ISMINE_SPENDABLE,
+    ISMINE_ENUM_ELEMENTS,
 };
 /** used for bitflags of isminetype */
 typedef uint8_t isminefilter;
 
 isminetype IsMine(const CKeyStore& keystore, const CScript& scriptPubKey);
 isminetype IsMine(const CKeyStore& keystore, const CTxDestination& dest);
 
+struct CachableAccount
+{
+    enum Type {
+        CREDIT,
+        DEBIT,
+    } m_type{CREDIT};
```

Comment:
Remove, it is initialized in the constructor. Also, should be const.

Reply:
Could be an `enum class`?

Reply:
Made `enum class` but not `const`. It doesn't look like 'const enum class' ever occurs anywhere else, so assuming it's not a thing you do..

Reply:
I mean `... const m_type`, meaning it can't change after initialization.

Reply:
Ah, I see.